installer/InstallBuilder/linuxdpkg.py

```diff
@@ -166,9 +166,11 @@ def BuildPackage(self):
             return
 
         # Build the package - 'cd' to the directory where we want to store result
-        dpkg_path = 'dpkg'
         if "DPKG_LOCATION" in self.variables:
             dpkg_path = self.variables["DPKG_LOCATION"]
+        # Use the system's version of dpkg-deb if present.
+        if os.path.exists('/usr/bin/dpkg-deb'):
```

Comment:
Don't you want this before line 169 to still allow the path to be configurable through environment variables?